recipes/libevent/2.1.11/conanfile.py

```diff
@@ -47,30 +48,14 @@ def source(self):
         extracted_folder = "libevent-release-{0}-stable".format(self.version)
         os.rename(extracted_folder, self._source_subfolder)
 
-        os.rename(os.path.join(self._source_subfolder, "CMakeLists.txt"),
-                  os.path.join(self._source_subfolder, "CMakeListsOriginal.txt"))
-        shutil.copy("CMakeLists.txt",
-                    os.path.join(self._source_subfolder, "CMakeLists.txt"))
-
-        # patch 'beta' to 'stable' because there is no git repository which cmake uses to determine stage name
+    def _patch_sources(self):
+        # patch "beta" to "stable" because there is no git repository which cmake uses to determine stage name
         # needed for 2.1.10, not needed for 2.1.11 so marked as non-strict
         tools.replace_in_file(os.path.join(self._source_subfolder, "cmake", "VersionViaGit.cmake"),
                               'set(EVENT_GIT___VERSION_STAGE "beta")',
                               'set(EVENT_GIT___VERSION_STAGE "stable")',
                               strict=False)
 
-    def imports(self):
-        # Copy shared libraries for dependencies to fix DYLD_LIBRARY_PATH problems
```

Comment:
this won't work. I complained about same thing, it's a limitation from libevent.

Reply:
The error is:
```
CMake Error at C:/Program Files/CMake/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
  system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY
  OPENSSL_INCLUDE_DIR)
```

So this is issue #419

Reply:
Using #486 and some libevent patches, I've got it working.

Reply:
This has been fixed using a patch.

Reply:
It won't work on macOS as stated in the comment in deleted chunk because it is the problem with DYLD_LIBRARY_PATH. Which patch fixed it?

Reply:
conftest binaries are files created by autotools which is not used by this recipe. CMake is used.

I patched the use of openssl in the cmake scripts to use the conan variables.

Reply:
Yes, you are right. Configure script were dropped in bincrafters/conan-libevent for 2.1.10 so the workaround for conftest can be safely dropped. Thanks for improving the recipe!

Reply:
Thanks for confirming!
I'm glad the recipe could be simplified!